data/helpers.d/psql

```diff
@@ -202,6 +200,10 @@ ynh_psql_setup_db() {
 	# If $db_pwd is not given, use new_db_pwd instead for db_pwd
 	db_pwd="${db_pwd:-$new_db_pwd}"
 
+	if [ $(ynh_psql_user_exists --user=$db_user) ]; then
+		ynh_psql_create_user "$db_name" "$db_user" "$db_pwd"
```

Comment:
There's 3 arguments given there ... but doesn't the helper only takes 2 ?

Reply:
You're right :-)